bindgen/ir/comp.rs

```diff
@@ -1422,7 +1423,9 @@ impl CompInfo {
 
                         // A declaration of an union or a struct without name
                         // could also be an unnamed field, unfortunately.
-                        if cur.spelling().is_empty() &&
+                        let mut spelling = cur.spelling();
```

Comment:
This is the bit I had missed in https://github.com/rust-lang/rust-bindgen/issues/2312#issuecomment-1281656605, thanks for catching it :)